3.18/bioc/kunpeng2/config.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# ===================================
+# Settings for kunpeng2 (Linux ARM64)
+# ===================================
+
+# Specific for openEuler START because some libraries are installed from source
```

Comment:
Environment variables like this don't control BBS and are not related to it either. It would be much better to define them in a place that makes them available to all users on your machine e.g. in `/etc/profile` (that's where we define them on our Linux and Mac builders).

Reply:
I already have them in the profile but I also put them here so that we have them in Git.
How about moving them to nodes/kunpeng2/local-settings.sh ?

Reply:
> How about moving them to nodes/kunpeng2/local-settings.sh ?

Yes, that's a better place to have them.